### PR TITLE
Task urgency update

### PR DESCRIPTION
**Pull request by grammeaway**

### CODE REVIEW

File: src/views/Task/Create.js [hunk 1]
```diff
@@ -153,6 +153,43 @@ export default withRouter(class Create extends Component {
               Deadline
               <DatePicker onChange={this.updateEndDate} minDate={this.state.startDate} selected={this.state.endDate} />
             </label>
+            <div>
+            <label>
+              Urgency
+              <br></br>
+              <label>
```
Comment (MadsDueKristensen) [inline src/views/Task/Create.js]: Indentation not right, from here and down.

File: src/views/Task/Update.js [hunk 2]
```diff
@@ -182,6 +182,43 @@ export default withRouter(class Update extends Component {
               Deadline
             <DatePicker onChange={this.updateDeadline} minDate={this.state.start} selected={this.state.deadline} />
             </label>
+            <div>
+            <label>
+              Urgency
+              <br></br>
+              <label>
+              <input
```
Comment (MadsDueKristensen) [inline src/views/Task/Update.js]: Indentation not right, from here and down.

File: src/views/Task/Create.js [hunk 3]
```diff
@@ -6,22 +6,22 @@ import moment from 'moment'
...
+import './Task.css'
 
 export default withRouter(class Create extends Component {
   state = {
     startDate: moment(),
     endDate: moment(),
     title: "",
     description: "",
-    urgency: "",
+    urgencystring: 'norush',
```
Comment (alex6296) [inline src/views/Task/Create.js]: i can see you use norush as a validater later maeby add a coment to specify what is means?

Comment (alex6296) [reply src/views/Task/Create.js]: ican see it says norush now :D cammelcase meaby?

File: src/views/Task/Update.js [hunk 4]
```diff
@@ -182,9 +182,46 @@ export default withRouter(class Update extends Component {
               Deadline
             <DatePicker onChange={this.updateDeadline} minDate={this.state.start} selected={this.state.deadline} />
             </label>
+            <div>
+              <label>
+                Urgency
+              <br></br>
```
Comment (alex6296) [inline src/views/Task/Update.js]: if i understood it corectly <br> should be done with divs and css :)

Comment (alex6296) [reply src/views/Task/Update.js]: the br tag*

File: src/views/Task/Create.js (hunk 3 above)
Comment (grammeaway) [reply src/views/Task/Create.js]: Will add explanatory comments, has to be lowercase for serverside reasons 🌮

File: src/views/Task/Create.js [hunk 5]
```diff
@@ -6,22 +6,22 @@ import moment from 'moment'
...
+import './Task.css'
 
 export default withRouter(class Create extends Component {
   state = {
     startDate: moment(),
     endDate: moment(),
     title: "",
     description: "",
-    urgency: "",
+    urgencystring: 'norush', //Basic value in raio togglegroup, setting it ensures something is selected
```
Comment (kvartborg) [inline src/views/Task/Create.js]:
```suggestion
    urgencystring: 'norush', // Basic value in radio togglegroup, setting it ensures something is selected
```

File: src/views/Task/Create.js [hunk 6]
```diff
@@ -153,6 +153,44 @@ export default withRouter(class Create extends Component {
               Deadline
               <DatePicker onChange={this.updateEndDate} minDate={this.state.startDate} selected={this.state.endDate} />
             </label>
+            <div>
+              <label>
+                Urgency
+              <div></div>
```
Comment (kvartborg) [inline src/views/Task/Create.js]:
```suggestion
```

File: src/views/Task/Create.js [hunk 7]
```diff
@@ -187,6 +187,44 @@ export default withRouter(class Create extends Component {
...
+               Urgency
+              <div></div>
+              <label>
+                <input
+                  type="radio"
+                  name="urgencystring"
+                  value="norush"
+                  onChange={this.handleChange}
+                  checked={this.state.urgencystring === 'norush'}
+                  required />
```
Comment (kvartborg) [inline src/views/Task/Create.js]:
```suggestion
                  required 
                />
```

File: src/views/Task/Create.js [hunk 8]
```diff
@@ -187,6 +187,44 @@ export default withRouter(class Create extends Component {
...
+              </label>
+              <span></span>
+              <label>
+                <input
+                  type="radio"
+                  name="urgencystring"
+                  value="urgent"
+                  onChange={this.handleChange}
+                  checked={this.state.urgencystring === 'urgent'}
+                  required />
```
Comment (kvartborg) [inline src/views/Task/Create.js]:
```suggestion
                  required 
                />
```

File: src/views/Task/Create.js [hunk 9]
```diff
@@ -187,6 +187,44 @@ export default withRouter(class Create extends Component {
...
+              </label>
+              <span></span>
+              <label>
+                <input
+                  type="radio"
+                  name="urgencystring"
+                  value="asap"
+                  onChange={this.handleChange}
+                  checked={this.state.urgencystring === 'asap'}
+                  required />
```
Comment (kvartborg) [inline src/views/Task/Create.js]:
```suggestion
                  required 
                />
```

File: src/views/Task/Create.js (hunk 6 above)
Comment (grammeaway) [reply src/views/Task/Create.js]: Missaligns aggresively without it hombre

Comment (grammeaway) [reply src/views/Task/Create.js]: Would you rather have <br>?

**Pull request merged**